src/snowflake/cli/api/project/schemas/streamlit/streamlit.py

```diff
@@ -44,3 +44,4 @@ class Streamlit(UpdatableModel, ObjectIdentifierModel(object_name="Streamlit")):
     title: Optional[str] = Field(
         title="Human-readable title for the Streamlit dashboard", default=None
     )
+    comment: Optional[str] = Field(title="Comment for the Streamlit app", default=None)
```

Comment:
This change will not take effect as this is V1 implementation. Do the changes in
https://github.com/snowflakedb/snowflake-cli/blob/2f54589ce681512d59684dd47ce89edc6d7c5b86/src/snowflake/cli/api/project/schemas/entities/streamlit_entity_model.py#L29